script/storybook-deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+if [[ $( git diff --name-only HEAD~1..HEAD webpack/ package.json | wc -l ) -ne 0 ]]; then
```

Comment:
Won't this break if a PR has multiple commits? Maybe it's safer to use `$TRAVIS_COMMIT_RANGE`. Note it can be empty so you should check for that.